x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_chart_distribution.js

```diff
@@ -654,10 +716,47 @@ export class ExplorerChartDistribution extends React.Component {
     const isLoading = seriesConfig.loading;
 
     return (
-      <div className="ml-explorer-chart" ref={this.setRef.bind(this)}>
-        {isLoading && <LoadingIndicator height={CONTENT_WRAPPER_HEIGHT} />}
-        {!isLoading && <div className="content-wrapper" />}
-      </div>
+      <>
+        <RuleEditorFlyout
+          setShowFunction={this.setShowRuleEditorFlyoutFunction}
```

Comment:
Not to be addressed in this PR but I think we should move away from this pattern of having a child set a parent component setter since it's an antipattern. I know this is old code and this is the way it's done in the Rule editor flyout currently but just think we should add this to the list of things to improve.